WordPress/openverse
# Make it easier to run Openverse from an offline media

## Fixes



Fixes #4478 by @dhruvkb

## Description



This PR makes Openverse easier to install from an offline media. It does so by providing recipes that can

- clean all env files from the repo to make sure that are no more secrets:  `just unenv`
- dump all Docker images needed for Openverse into `.tar.gz` files: `just docker/dump-3p && just docker/dump-lint && just docker/dump-ov`
- load all Docker images from the images directory: `just docker/load`.

## Testing Instructions


0. Dump all Docker images using the command above.
1. Clear your Docker images using `docker image prune -a`.
2. Load the images back using the command above.
3. Run Openverse, it should not need to download anything from the Internet.

## Checklist


      PRs) or the media properties generator (`just catalog/generate-docs media-props`
      for the catalog or `just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
